elections/steering/2022/election.yaml

```diff
@@ -0,0 +1,17 @@
+name: 2022 Steering Committee Election
+organization: Kubernetes
+start_datetime: 2022-09-06 00:00:01
+end_datetime: 2022-09-29 23:59:00
```

Comment:
@jberkus @coderanger I don't know how to specify the AoE time

Reply:
https://www.timeanddate.com/worldclock/converter.html?iso=20220930T115900&p1=tz_aoe&p2=1440

Shows 11:59:59 on Sep 30, 2022. will go with that.